### PR TITLE
Theme improvements

### PR DESCRIPTION
**Pull request by isholgueras**
Added basic styles for forms.

### CODE REVIEW

File: web/themes/dcamp_2017_theme/scss/components/_forms.scss [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
-  input[type="submit"]{
-    @include button();
-    padding:10px;
-  }
-}
\ No newline at end of file
+//form{
+//  input[type="text"],
+//  input[type="password"],
+//  input[type="email"],
```
Comment (juampynr) [inline web/themes/dcamp_2017_theme/scss/components/_forms.scss]: @isholgueras shall we delete this file?

File: web/themes/dcamp_2017_theme/scss/layout/_content.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+//.region-content {
+//  max-width: $max-width;
```
Comment (juampynr) [inline web/themes/dcamp_2017_theme/scss/layout/_content.scss]: Kill it, daddy. :eyeglasses:

File: web/themes/dcamp_2017_theme/scss/components/_forms.scss (hunk 1 above)
Comment (isholgueras) [inline web/themes/dcamp_2017_theme/scss/components/_forms.scss]: @juampynr I suppose we should, yes.

File: web/themes/dcamp_2017_theme/scss/layout/_content.scss (hunk 2 above)
Comment (isholgueras) [inline web/themes/dcamp_2017_theme/scss/layout/_content.scss]: That's my common mistake. Sure I will.

**Comment by juampynr** [comment]
@isholgueras please create a new PR against this repo instead of your fork so I can give you ahand with the conflicts and with removing commented code.

**Comment by juampynr** [comment]
Closing. @isholgueras, please create a new PR directly against the repository.

**Pull request closed without merging**